masonry/src/widgets/checkbox.rs

```diff
@@ -95,18 +101,13 @@ impl Widget for Checkbox {
         match event {
             PointerEvent::Down { .. } => {
                 ctx.capture_pointer();
-                // Checked state impacts appearance and accessibility node
-                ctx.request_render();
```

Comment:
I have not checked this in detail, but is this safe to do? On a skim, `capture_pointer` results in `widget_state.is_active == true`, which results in a different paint state.

Reply:
I believe so, because the "active" change is handled in `fn update`. And indeed, I can see that it works - holding down the button makes the background change as expected.